easybuild/tools/package/utilities.py

```diff
@@ -121,15 +129,26 @@ def package_with_fpm(easyblock):
         '--iteration', pkgrel,
         '--description', quote_str(easyblock.cfg["description"]),
         '--url', quote_str(easyblock.cfg["homepage"]),
+    ]
+    excludes = ['--exclude %s' % quote_str(x) for x in exclude_files_glob]
```

Comment:
collapse this with the 2nd `exclude_files_glob =`:

```python
excludes = ['--exclude %s' % quote_str(os.path.join(self.installdir[1:], x)) for x in exclude_files_glob]
```

Still not sure what the `[1:]` is about